web/website/content/_index.md

```diff
@@ -169,8 +169,8 @@ showcase_section:
         SELECT
           *,
           started + unfinished AS finished,
-          (started + unfinished) / started AS fin_share,
-          (started + unfinished) / started / (1 - (started + unfinished) / started) AS fin_ratio
+          (started + unfinished * 1.0 / started) AS fin_share,
```

Comment:
This produces incorrect results; it's a manifestation of https://github.com/PRQL/prql/pull/2684#discussion_r1213656459.

We could use parentheses; we'll get too many, but that might be OK. Particularly if we only do this on explicit dialects.